lib/msal-browser/src/index.ts

```diff
@@ -72,6 +72,8 @@ export {
     // Utils
     StringUtils,
     UrlString,
+    // AzureCloudInstance enum
+    AzureCloudInstance,
```

Comment:
Do we want to export `AzureCloudOptions` as well?

Reply:
Yeah, guess we have to. For types. Adding.